server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -147,6 +152,12 @@ public NodeStats(StreamInput in) throws IOException {
         } else {
             indexingPressureStats = null;
         }
+        if (ShardIndexingPressureSettings.isShardIndexingPressureAttributeEnabled()) {
```

Comment:
Same curiosity here: `if (in.getVersion().onOrAfter(Version.V_1_1_0)) { ` ?

Reply:
same as above.

Reply:
:+1: all set, thank you for explaning